ddp-workspace/projects/sandbox-app/src/app/sandboxes/activityForm/addressEmbeddedSandbox.component.ts

```diff
@@ -1,9 +1,9 @@
-import { Component, OnInit, ViewChild, OnDestroy } from '@angular/core';
-import { UserActivityServiceAgent, AddressEmbeddedComponent, AddressService, CompositeDisposable } from 'ddp-sdk';
-import { Observable, BehaviorSubject } from 'rxjs';
+import { Component, OnDestroy, OnInit, ViewChild } from '@angular/core';
+import { AddressEmbeddedComponent, AddressService, CompositeDisposable, UserActivityServiceAgent } from 'ddp-sdk';
+import { BehaviorSubject, empty } from 'rxjs';
 import { mergeMap } from 'rxjs/operators';
-import { empty } from 'rxjs';
 import * as _ from 'underscore';
+import { MailAddressBlock } from '../../../../../ddp-sdk/src/lib/models/activity/MailAddressBlock';
```

Comment:
```suggestion
import { MailAddressBlock } from 'ddp-sdk';
```
And probably we have to update `public-api.ts`